test/api/handlers_test.go

```diff
@@ -280,14 +280,53 @@ func testUserPUT(t *testing.T, at *test.AccountsTester) {
 		t.Fatalf("Expected the user to have StripeID %s, got %s", stripeID, u2.StripeID)
 	}
 	// Try to update the StripeID again. Expect this to fail.
-	r, _, err := at.UserPUT("", stripeID)
+	r, _, err := at.UserPUT("", "", stripeID)
 	if err == nil || !strings.Contains(err.Error(), "409 Conflict") || r.StatusCode != http.StatusConflict {
 		t.Fatalf("Expected to get error '409 Conflict' and status 409, got '%s' and %d", err, r.StatusCode)
 	}
 
+	// Update the user's password with an empty one. Expect this to succeed but
```

Comment:
does the comment match the code here? we expect a 400 right?

Reply:
You're right. I'll F/U in https://github.com/SkynetLabs/skynet-accounts/pull/100